x-pack/plugins/spaces/common/types.ts

```diff
@@ -21,3 +22,18 @@ export type GetAllSpacesPurpose =
 export interface GetSpaceResult extends Space {
   authorizedPurposes?: Record<GetAllSpacesPurpose, boolean>;
 }
+
+export interface GetShareableReferencesResponse {
+  /** The count of references that are not tags. */
+  relativesCount: number;
+  /** The count of references that are tags. */
+  tagsCount: number;
```

Comment:
Can we get around this tag dissociation by having the response returning the count per type, and let the UI decide how they want to handle/aggregate these per-type results?

Reply:
Yeah, especially if we hard-code the `'tag'` type in the UI as you suggested before.

Reply:
I updated this to just pass the results of `collectMultiNamespaceReferences` to the client, the client can do any post processing on its own.